library/src/nvcc_detail/hipsparse.cpp

```diff
@@ -11466,6 +11470,125 @@ hipsparseStatus_t hipsparseSpGEMM_copy(hipsparseHandle_t      handle,
 }
 #endif
 
+#if(!defined(CUDART_VERSION) || CUDART_VERSION >= 11000)
+HIPSPARSE_EXPORT
+hipsparseStatus_t hipsparseSpGEMMreuse_workEstimation(hipsparseHandle_t      handle,
```

Comment:
I think spgemm reuse only exists in cusparse 11.3.1 and greater. This will fail to compile with cusparse backend I think

Reply:
confirmed! changing this right away